addons/stock/report/report_deliveryslip.xml

```diff
@@ -94,9 +94,12 @@
                                         <span t-field="move_line.product_id.sudo().description_pickingin"/>
                                     </p>
                                 </td>
-                                <td t-if="has_serial_number" groups="stock.group_lot_on_delivery_slip">
+                                <td t-if="has_serial_number and move_line.lot_name" groups="stock.group_lot_on_delivery_slip">
                                     <span t-field="move_line.lot_name"/>
                                 </td>
+                                <td t-else="" groups="stock.group_lot_on_delivery_slip">
```

Comment:
shouldn't it be t-if move_line.lot_id ?
i think move_line.lot_id.name will crash if lot_id is an empty recordset, right?

Reply:
@sle-odoo No it is ok, it will just return False